shared/monitor/monitor.py

```diff
@@ -67,7 +67,8 @@ def get_results():
     if 'kafka-sub' in j:
       sub = j['kafka-sub']
       kafka_msg = ' This data is also being published to EventStreams (kafka). Subscribe with: <p style="font-family:monospace;">' + sub + '</p>\n'
-    return render_template("monitor.html", **locals())
+    context = {'s': s, 'n': n, 'c': c, 'ct': ct, 'it': it, 'u': u, 'kafka_msg': kafka_msg }
```

Comment:
In the future, might be good to figure out if there's a more automatic way to convert Python variables into a dictionary of variable name and value like this.